paymentsheet/api/paymentsheet.api

```diff
@@ -45,13 +45,33 @@ public final class com/stripe/android/paymentsheet/ComposableSingletons$PaymentO
 	public final fun getLambda-2$paymentsheet_release ()Lkotlin/jvm/functions/Function2;
 }
 
+public abstract interface class com/stripe/android/paymentsheet/CreateIntentCallback : com/stripe/android/paymentsheet/AbsCreateIntentCallback {
```

Comment:
Can we get rid of `AbsCreateIntentCallback`?

Reply:
We’ll eventually need it again when we add an integration type that doesn’t use `PaymentMethod`. You want me to remove it until then?

Reply:
I think so, we're unintentionally leaking it as an implementation detail right now. I'd think we could do it without leaking our implementation details when we need it